components/Content/__stories__/Content.stories.js

```diff
@@ -40,6 +43,7 @@ storiesOf('Content', module)
     'with one column',
     withInfo()(() => (
       <Content
+        backgroundImageSource={text('imageSource', backgroundImage)}
```

Comment:
I like leaving the story to match up with default props. Also the knob name should match the prop name.

If you would please remove line 11 and then make the other three diffs on this page match this:
```
backgroundImageSource={text('backgroundImageSource', '')}
```

Reply:
Okay, updating to that. FYI though, HeroBanner.stories.js doesn't have correctly named knob then, it also uses `imageSource` when prop is named `backgroundImageSource`.

Reply:
Damn. Would you mind fixing that here?